doc/src/manual/strings.md

```diff
@@ -516,9 +514,8 @@ false
 julia> contains("Xylophon", 'o')
 ERROR: MethodError: no method matching contains(::String, ::Char)
 Closest candidates are:
-  contains(!Matched::Function, ::Any, !Matched::Any) at reduce.jl:640
-  contains(::AbstractString, !Matched::AbstractString) at strings/search.jl:366
- ...
+  contains(::Function, ::Any, ::Any) at reduce.jl:628
```

Comment:
So we want the `!Matched` notation? I can just update and run without color. It would be neat if Documenter could color these as well. Perhaps by matching `!Matched` and color the following type.

Anyhow, I can update this PR accordingly

```
julia> bar(1.)
ERROR: MethodError: no method matching bar(::Float64)
Closest candidates are:
  bar(!Matched::Int64) at REPL[1]:1
```